docs/release-notes/8.0.asciidoc

```diff
@@ -8,3 +8,35 @@
 [discrete]
 [[known-issues-8.0.0-rc2]]
 ==== Known issues
+
+*The Data View option might not display in upgraded environments with legacy alerts:*
+
+NOTE: If new alerts are generated in an upgraded environment without legacy alerts, refreshing any page in {es-sec} will make the *Data View* option appear in the {es-sec-ui}.
+
+After upgrading, the *Data View* option might not display in environments with legacy alerts. To resolve this, a user with elevated role privileges must visit the {es-sec-app}, open a page that displays alert data (such as the Overview page), and refresh the page ({pull}121390[#121390]).
+
+The role must have the following privileges:
+
+* *Cluster privileges*: The `manage` privilege
+* *Index privileges*: The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices where `<kib-space>` is the {kib} space name:
+
+** `.siem-signals-<kib-space>`
+** `.lists-<kib-space>`
+** `.items-<kib-space>`
+** `.alerts-security.alerts-<kib space>`
+** `.internal.alerts-security.alerts-<kib space>-*`
+
+* *{kib} space*: `All` privileges for the `Security` feature (see
+{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
+
+*Rules must be manually re-enabled after upgrading:*
+
+Rules are automatically disabled when upgrading to {stack} version 8.0.0 and must be manually re-enabled after completing the upgrade process. Not doing this could cause a gap in rule coverage ({pull}120906[#120906]).
+
+NOTE: If you're using the {es-sec} API to re-enable rules, refer to <<bulk-actions-rules-api, Bulk rule actions>>.
+
+To re-enable your rules:
+
+. Go to the All rules table on the Rules page (*Detect -> Rules*).
+. Click the checkbox in the *Rule* column to select all rules.
```

Comment:
I think selecting this checkbox will only select the rules that are currently displayed on the page (20 rows by default). There's a separate UI element for "Select all ### rules".

Reply:
@jmikell821 and @benironside , @joepeeples noted that [in our docs](https://www.elastic.co/guide/en/security/current/rules-ui-management.html#select-all-prebuilt-rules-white_check_mark) that we use “click **Select all (number) rules**”. WDTY about continuing to use this when referring to the the **Select all rules** button on the Rules table?